Dockerfile

```diff
@@ -33,26 +52,19 @@ ENV NTA_DATA_PATH /usr/local/src/nupic/prediction/data
 # OPF needs this
 ENV USER docker
 
-# Copy context into container file system
-ADD . $NUPIC
-
-WORKDIR /usr/local/src
-
-# Clone nupic.core
-RUN git clone `head -n 2 $NUPIC/.nupic_modules | tail -n 1 | sed -r "s/NUPIC_CORE_REMOTE = '(.+)'/\\1/"` && \
-    cd nupic.core && \
-    git reset --hard `head -n 3 $NUPIC/.nupic_modules | tail -n 1 | sed -r "s/NUPIC_CORE_COMMITISH = '(.+)'/\\1/"`
-
+# Set up nupic.core
+RUN pip install numpy pycapnp
```

Comment:
> In particular, install pip, setuptools, and wheel like this: https://github.com/numenta/nupic.core/blob/master/Dockerfile#L18-L20

See above, those three lines are there already: https://github.com/numenta/nupic/pull/3314/files#diff-3254677a7917c6c01f55212f86c57fbfR40



> Also consider not building nupic.core from source and instead install nupic.bindings from binary wheel. Recent changes to setup.py allow for correct handling of pycapnp and other dependencies (like numpy), so you shouldn't need RUN pip install numpy pycapnp and subsequent lines.

I also agree with installing nupic.core from pypi in the dockerfile, but let's do that in another PR.